### PR TITLE
Bug/6972 fix duplicate urls

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue:

- #6972

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const Template = ( { setupRegistry = () => {}, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<WidgetWithComponentProps { ...args } />
+	</WithRegistrySetup>
+);
+
+export const Ready = Template.bind( {} );
+Ready.storyName = 'Ready';
+Ready.args = {
+	setupRegistry: setupSearchConsoleAnalyticsMockReports,
```
Comment (aaemnnosttv) [inline assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js:43]: This component doesn't use data from Search Console so it isn't clear whether this is intentional or not. Can we use something specific for GA here instead?

File: assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+export const GatheringData = Template.bind( {} );
+GatheringData.storyName = 'GatheringData';
+GatheringData.args = {
+	setupRegistry: setupAnalyticsGatheringData,
+};
+
+export const ZeroData = Template.bind( {} );
+ZeroData.storyName = 'Zero Data';
+ZeroData.args = {
+	setupRegistry: setupSearchConsoleAnalyticsZeroData,
```
Comment (aaemnnosttv) [inline assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js:55]: Same here.

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @eugene-manuilov this looks good, just one question but also, VRT is failing.

File: assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js (hunk 1 above)
Comment (eugene-manuilov) [reply assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js:43]: Good point. Updated.

File: assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js (hunk 2 above)
Comment (eugene-manuilov) [reply assets/js/components/wp-dashboard/WPDashboardPopularPages.stories.js:55]: Yep, updated.

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks @eugene-manuilov !

**Pull request merged**